README.md

```diff
@@ -319,6 +328,20 @@ Note that linking against Charm++ dynamic libraries produces much smaller
 binaries and takes much less linking time.
 
 
+## Contributing
+
+The recommended way to contribute to Charm++ development is to open a pull request (PR) on GitHub.
+To open a pull request, create a fork of the Charm++ repo in your own space
+(if you already have a fork, make sure is it up-to-date), and then create a new branch off of the
+`main` branch.
+
+GitHub provides a detailed tutorial on creating pull requests 
+(https://docs.github.com/en/pull-requests/collaborating-with-pull-requests). 
+
+Our wiki also contains additional information about pull requests
+(https://github.com/UIUC-PPL/charm/wiki/Working-with-Pull-Requests).
+
+
```

Comment:
Could maybe add something about how PRs need to go through code review and pass CI testing before they can be merged by someone on the Core development team. Otherwise LGTM

Reply:
Thanks, added in 1fbce231d1a2735d7fd9e82d36881eb834851eff